tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -149,6 +149,13 @@ var _ = Describe("odo devfile create command tests", func() {
 		})
 	})
 
+	Context("When executing odo create with component with no devBuild command", func() {
+		It("should successfully create the devfile component", func() {
+			// Quarkus devfile has no devBuild command
+			helper.CmdShouldPass("odo", "create", "quarkus")
```

Comment:
Please assert some output statement to confirm the pass.

Reply:
@prietyc123 Updated. There's no specific output in the `odo create` output that says quarkus was created:
```
Johns-MacBook-Pro-3 johncollier$ odo create quarkus
Experimental mode is enabled, use at your own risk

Validation
 ✓  Checking devfile compatibility [145826ns]
 ✓  Creating a devfile component from registry: DefaultDevfileRegistry [112673ns]
 ✓  Validating devfile component [398706ns]

Please use `odo push` command to create the component with source deployed
```

So I just checked for `Please use 'odo push' command to create the component with source deployed`

Reply:
> So I just checked for Please use 'odo push' command to create the component with source deployed

make sense, thanks 👍